commit 9b947487fb38dec60f5d56242c8b50e21a923ba3
Author: pmelange <isprotejesvalkata@gmail.com>
Date:   Thu Nov 15 22:32:51 2018 +0100

    luci: add new dependency to luci-lib-ipkg
    
    This became an issue because the luci-lib-ipkg code has been
    removed from luci-base.  The upstream commit is
    https://github.com/openwrt/luci/commit/4bbe32548c407ec6ae104934b3f1fe53539de9f6

diff --git a/utils/luci-app-ffwizard-berlin/Makefile b/utils/luci-app-ffwizard-berlin/Makefile
index b7fa19f9d..dd73aa651 100644
--- a/utils/luci-app-ffwizard-berlin/Makefile
+++ b/utils/luci-app-ffwizard-berlin/Makefile
@@ -16,7 +16,7 @@ define Package/luci-app-ffwizard-berlin
   SUBMENU:=3. Applications
   TITLE:=Freifunk Berlin configuration wizard
   URL:=http://berlin.freifunk.net
-  DEPENDS+= +luci-mod-admin-full +freifunk-policyrouting +luci-lib-jsonc +community-profiles
+  DEPENDS+= +luci-mod-admin-full +freifunk-policyrouting +luci-lib-jsonc +community-profiles +luci-lib-ipkg
   PKGARCH:=all
 endef
 
diff --git a/utils/luci-mod-freifunk-ui/Makefile b/utils/luci-mod-freifunk-ui/Makefile
index e943501e8..2d49ecbe6 100644
--- a/utils/luci-mod-freifunk-ui/Makefile
+++ b/utils/luci-mod-freifunk-ui/Makefile
@@ -20,7 +20,7 @@ define Package/luci-mod-freifunk-ui
   CATEGORY:=LuCI
   SUBMENU:=2. Modules
   TITLE:=Freifunk Public and Admin LuCI UI
-  DEPENDS:=+luci-mod-admin-full +luci-lib-json +community-profiles
+  DEPENDS:=+luci-mod-admin-full +luci-lib-json +community-profiles +luci-lib-ipkg
   PKGARCH:=all
 endef
 
